### DISCUSSION TITLE
Is there a way to configure the default keyring from the pyproject.toml file?

### DISCUSSION THREAD

**Discussion by mdlam92** [python-poetry/poetry#7152] [Q&A]
To skirt https://github.com/python-poetry/poetry/issues/1917 on the many systems I deploy my repo to, can I configure the default keyring poetry uses from the toml file?

**Comment by anoduck** [comment]
Probably a brilliant feature request, but since the credentials part is experimental, it does not appear this has been mapped out yet. As it is not listed in the configuration documentation: https://python-poetry.org/docs/master/configuration/
